src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
```

Comment:
I think this breaks the conventions in the project.
1. The resource is named `GroupsRoute` but the file is `group.route.ts`. I would ask to rename the file, as this route handles `groups` requests not just one group.
2. All other imports are path alias imports and not relative imports. So, the import should be:
```import GroupsRoute from '@routes/group.route';```